PHP/Button/Getting-Started.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Getting-Started
-description: getting started
+title: Getting Started | JavaScript | Button | Syncfusion
+description: Learn here more about getting started with Syncfusion Essential JavaScript Button Control, its elements, and more.
 platform: js
 control: Button
 documentation: ug
 ---
 
-# Getting Started
+# Getting Started with JavaScript Button
```

Comment:
Check and change in all places [change PHP instead of JavaScript]

Reply:
Done